src/components/header/Header.tsx

```diff
@@ -96,32 +95,92 @@ const Header: React.FC<Props> = (_): JSX.Element => {
     [items, activeKey, headerHeight]
   )
 
-  const color = useMemo(() => palette('text', 0)({ theme }), [theme])
+  const links = items.map(({ label, key, path, icon: Icon }) => (
+    <Link key={key} to={path} onClick={closeMenu}>
+      <HeaderDrawerItem>
+        <Icon style={{ marginLeft: '12px', marginRight: '12px' }} />
+        {label}
+      </HeaderDrawerItem>
+    </Link>
+  ))
+
   const iconStyle = { fontSize: '1.5em', marginRight: '20px' }
+  const color = useMemo(() => palette('text', 0)({ theme }), [theme])
 
   return (
     <HeaderContainer>
-      <Row justify="space-between" align="middle" style={{ height: headerHeight }}>
-        <Col>
-          <Row justify="space-between" align="middle" style={{ height: headerHeight }}>
-            <AsgardexLogo />
-            <HeaderNetStatus />
-          </Row>
-        </Col>
-        <Col span="auto">
-          <Row justify="center" align="bottom" style={{ height: headerHeight }}>
-            <Tabs activeKey={activeKey}>{tabs}</Tabs>
-          </Row>
-        </Col>
-        <Col>
-          <Row align="middle">
-            <ThemeIcon onClick={clickSwitchThemeHandler} style={{ color, ...iconStyle }} />
-            <SettingsIcon onClick={clickSettingsHandler} style={{ color, ...iconStyle }} />
-            <LockIcon onClick={clickLockHandler} style={{ ...iconStyle }} />
+      <>
+        <Row justify="space-between" align="middle" style={{ height: headerHeight }}>
+          <Col xs={0} sm={0} md={0} lg={24}>
```

Comment:
[nitpick] Just `xs={0} lg={24}` should be fine (not need to set other breakpoints w/ same values).